src/scripts/OSFramework/OSUI/Pattern/TimePicker/scss/_timepicker.scss

```diff
@@ -24,6 +24,11 @@
 			}
 		}
 
+		// Hide the platform input which is set as hidden by the library and we're change it into the expected type, however we do not want it visible since library will add a clone to better deal with the selected dates.
+		&:first-of-type {
```

Comment:
instead of the !important, we should override the rules setting the inline-block, by increasing the specificity:

```
.chrome input[type=time], 
.chrome input[type=time]:empty
```

Reply:
Already mentioned above.